docs/quickstart.rst

```diff
@@ -144,3 +144,140 @@ Otherwise, models are represented as dicts.
 
     Values in ``request.swagger_data`` are only available if
     ``pyramid_swawgger.enable_request_validation`` is enabled.
+
+Accessing Swagger Operation
+---------------------------
+
+During the implementation of an endpoint you could eventually have need of accessing the Swagger Specs that defined that specific view.
+``pyramid_swagger`` will inject in the request object a new property (that will be evaluated only if accessed) called ``operation``.
+
+``request.operation`` will be set to ``None`` for Swagger 1.2 defined endpoints, while it will be an `Operation <https://github.com/Yelp/bravado-core/blob/v4.8.4/bravado_core/operation.py#L12>`_ object if the endpoint is defined by Swagger 2.0 specs.
```

Comment:
Oof, I need to write the necessary PR to properly remove 1.2 support :( It has become a notable burden for us. (no action needed, just noting)